domains/Settings/SupportHome/index.js

```diff
@@ -0,0 +1,117 @@
+      StoreReview.requestReview();
+    }
+  };
+  const inputs = [
+    {
+      key: 'feedback',
+      label: I18n.t('supportHome.feedback'),
+      button: true,
+      touchable: false,
+      action: null
+    },
+    {
+      key: 'rateApp',
+      label: I18n.t('supportHome.rateApp'),
+      button: false,
+      touchable: true,
+      action: rateApp
+    }
+  ];
+  return (
+    <View>
+      {settingsView === 'Support' && supportView === '' && (
+        <View>
+          <View style={{ flexDirection: 'row', marginLeft: 'auto', marginRight: 'auto' }}>
+            <View style={{ paddingRight: '5%' }}>
+              <Button onPress={() => setSettingsView('Settings')}>{I18n.t('accountSettings.settings')}</Button>
+            </View>
+            <View style={{ paddingLeft: '5%' }}>
+              <Button mode="contained">
+                {I18n.t('accountSettings.support')}
+              </Button>
+            </View>
+          </View>
+          <View style={{ paddingLeft: '5%', paddingRight: '5%', paddingTop: 20 }}>
+            <Headline style={{ fontWeight: 'bold' }}>{I18n.t('supportHome.helpCenter')}</Headline>
+            <View style={styles.horizontalLineGray} />
+            {inputs.length > 0 && inputs.map((input) => (
+              <View key={input.key}>
+                {input.touchable ? (
+                  <TouchableOpacity style={{ flexDirection: 'row' }} onPress={() => input.action()}>
+                    <Text style={styles.text}>{input.label}</Text>
+                    {input.button && (
+                      <IconButton
+                        icon="chevron-right"
+                        size={30}
+                        color={theme.colors.primary}
+                        style={{ marginLeft: 'auto', marginTop: -5, marginBottom: -10 }}
+                        onPress={() => {
+                          setSupportView(input.key);
+                        }}
+                      />
+                    )}
+                  </TouchableOpacity>
+                ) : (
+                  <View style={{ flexDirection: 'row' }}>
+                    <Text style={styles.text}>{input.label}</Text>
+                    {input.button && (
+                    <IconButton
+                      icon="chevron-right"
+                      size={30}
+                      color={theme.colors.primary}
+                      style={{ marginLeft: 'auto', marginTop: -5, marginBottom: -10 }}
+                      onPress={() => {
+                        setSupportView(input.key);
+                      }}
+                    />
+                    )}
+                  </View>
+                )}
+                <View style={styles.horizontalLineGray} />
+              </View>
+            ))}
+          </View>
+          <Button onPress={() => {
+            setView(prevView);
+          }}
+          >
+            {I18n.t('accountSettings.back')}
+          </Button>
```

Comment:
What's the point of `prevView`?

I think you can treat the settings button more like a toggle button (each press either pops up the settings page or closes it). 

Or if you want to keep your current approach, the back buttons need to be working (they didn't work for me) when I wanted to go back to the main data collection page